#933 readme about cypress tests and snapshot update (#934)
tl-its-umich-edu/my-learning-analytics: README.md

 
 `npm run cypress:open`
 
 When running tests do not use the All Tests button due to unsolved issues. 
 Run the cypress test if UI change are there as part of the work.

>  If a snapshot fails due to change in the UI, try update the snapshot from the failed test from cypress controlled 

Reviewer: ```suggestion
 If a snapshot fails due to change in the UI, try updating the snapshot from the failed test from cypress controlled 
```

>  browser 'compare snapshot' and a pop up appears saying that `Upadate Shanpshot` 

Reviewer: ```suggestion
 browser 'compare snapshot' and a pop up appears to `Update Snapshot` 
```
